Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2831,6 +2831,7 @@ extension BrowserViewController: NewTabPageDelegate {
         if Preferences.General.isNewRetentionUser.value == true,
             Preferences.DebugFlag.skipNTPCallouts != true,
             !topToolbar.inOverlayMode,
+            topToolbar.currentURL == nil,
```

Comment:
Didnt have this already very weird

Reply:
Is it just me? OR is this somehow constantly being removed from development?
I have no idea why it was missing again.. but it seems this PR adds it back lol

Reply:
I agree this is totally weird :)